src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -22,4 +24,32 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object object) {
+        if (object == null) {
+            return false;
+        }
+
+        if (this == object) {
+            return true;
+        }
+
+        if (object.getClass().equals(Rectangle.class)) {
+            Rectangle figure = (Rectangle) object;
+            return Objects.equals(this.width, figure.width)
+                    && Objects.equals(this.length, figure.length)
+                    && Objects.equals(this.color, figure.color);
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/equals-and-hashcode/equals-and-hashcode

Reply:
Removed methods of Objects.class.